packages/lesswrong/components/sunshineDashboard/SunshineNewPostsList.jsx

```diff
@@ -43,7 +44,8 @@ const withListOptions = {
   collection: Posts,
   queryName: 'sunshineNewPostsListQuery',
   fragmentName: 'PostsList',
-  enableCache: true
+  enableCache: true,
+  enableTotal: true,
```

Comment:
Same comment as above for comments: performance is okay iff we've dealt with any years-old posts.